doc/website/blog/2023-02-23-simple-cms-with-ent.mdx

```diff
@@ -0,0 +1,777 @@
+```
+atlas migrate diff add_users_posts \
+  --dir "file://ent/migrate/migrations" \
+  --to "ent://ent/schema" \
+  --dev-url "docker://mysql/8/ent"
+```
+Observe that two new files were created:
+```
+ent/migrate/migrations
+|-- 20230226150934_add_users_posts.sql
+`-- atlas.sum
+```
+
+The SQL file (the actual file name will vary on your machine depending on the timestamp in which you run `atlas migrate diff`) contains the SQL DDL statements required to set up the database schema on an empty MySQL database:
+```sql
+-- create "users" table  
+CREATE TABLE `users` (`id` bigint NOT NULL AUTO_INCREMENT, `name` varchar(255) NOT NULL, `email` varchar(255) NOT NULL, `created_at` timestamp NOT NULL, PRIMARY KEY (`id`), UNIQUE INDEX `email` (`email`)) CHARSET utf8mb4 COLLATE utf8mb4_bin;  
+-- create "posts" table  
+CREATE TABLE `posts` (`id` bigint NOT NULL AUTO_INCREMENT, `title` varchar(255) NOT NULL, `body` longtext NOT NULL, `created_at` timestamp NOT NULL, `user_posts` bigint NULL, PRIMARY KEY (`id`), INDEX `posts_users_posts` (`user_posts`), CONSTRAINT `posts_users_posts` FOREIGN KEY (`user_posts`) REFERENCES `users` (`id`) ON UPDATE NO ACTION ON DELETE SET NULL) CHARSET utf8mb4 COLLATE utf8mb4_bin;
+```
+
+To setup our development environment, let's use Docker to run a local `mysql` container:
+```
+docker run --rm --name entdb -d -p 3306:3306 -e MYSQL_DATABASE=test -e MYSQL_ROOT_PASSWORD=pass mysql:8
+```
+
+Finally, let's run the migration script on our local database:
+```
+atlas migrate apply --dir file://ent/migrate/migrations \
+  --url mysql://root:pass@localhost:3306/test
+```
+Atlas reports that it successfully created the tables:
+```
+Migrating to version 20230220115943 (1 migrations in total):
+
+  -- migrating version 20230220115943
+    -> CREATE TABLE `users` (`id` bigint NOT NULL AUTO_INCREMENT, `name` varchar(255) NOT NULL, `email` varchar(255) NOT NULL, `created_at` timestamp NOT NULL, PRIMARY KEY (`id`), UNIQUE INDEX `email` (`email`)) CHARSET utf8mb4 COLLATE utf8mb4_bin;
+    -> CREATE TABLE `posts` (`id` bigint NOT NULL AUTO_INCREMENT, `title` varchar(255) NOT NULL, `body` longtext NOT NULL, `created_at` timestamp NOT NULL, `post_author` bigint NULL, PRIMARY KEY (`id`), INDEX `posts_users_author` (`post_author`), CONSTRAINT `posts_users_author` FOREIGN KEY (`post_author`) REFERENCES `users` (`id`) ON UPDATE NO ACTION ON DELETE SET NULL) CHARSET utf8mb4 COLLATE utf8mb4_bin;
+  -- ok (55.972329ms)
+
+  -------------------------
+  -- 67.18167ms
+  -- 1 migrations 
+  -- 2 sql statements
+
+```
+
+### Step 2: Seeding our database
+
+:::info
+
+The code for this step can be found in [this commit](https://github.com/rotemtam/ent-blog-example/commit/eae0c881a4edfbe04e6aa074d4c165e8ff3656b1).
+
+:::
+
+While we are developing our content management system, it would be sad to load a web page for our system and not see content for it. Let's start by seeding some data into our database and learn some Ent concepts.
```

Comment:
```suggestion
While we are developing our content management system, it would be sad to load a web page for our system and not see content for it. Let's start by seeding data into our database and learn some Ent concepts.
```